wls-wahlvorstand-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorstandservice/service/wahlvorstand/FunktionsnamenMapping.java

```diff
@@ -0,0 +1,14 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorstandservice.service.wahlvorstand;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class FunktionsnamenMapping {
```

Comment:
anstelle von 2 Maps könntest du auch eine Map verwenden die als Key `WahlbezirkArtModel` hat und die beiden Maps kapselt. Dann musst du für den Zugriff nicht erst eine neue temporäre Map bauen. In dem yml-File müssen entsprechend die Daten angepasst werden.

Reply:
siehe commit [01a4789](https://github.com/it-at-m/Wahllokalsystem/pull/533/commits/01a4789b65bcb24f634c3b1f96aee39f0f2cf44f)